src/problems/promise.js

```diff
@@ -0,0 +1,9 @@
+// Use the API 'https://omdbapi.com' to log in which the movie "titanic" was released.
+
```

Comment:
Please move this in the concepts file as well. This is not a standalone problem.

Reply:
made requested changes